a2agc/src/assets/pages/vis5-opioid-trajectories/visualization5.md

```diff
@@ -36,9 +33,7 @@ The visualization provides a comparison of the number of cases across each possi
 
 The table was created using the following SQL script:
 
-{{ include_sql('./data.sql') }}
-
-The resulting aggregated data can be found [here](assets/generated/visualization2/data.csv).
+The resulting aggregated data can be found [here].

```

Comment:
Why can't we link out to the sql?

Reply:
The links weren't working before but I think I just had the wrong urls. I'll add those back in.